pinot-broker/src/main/java/org/apache/pinot/broker/routing/RoutingManager.java

```diff
@@ -516,11 +526,14 @@ public Long getQueryTimeoutMs(String tableNameWithType) {
     // Time boundary manager is only available for the offline part of the hybrid table
     transient TimeBoundaryManager _timeBoundaryManager;
 
-    RoutingEntry(String tableNameWithType, SegmentPreSelector segmentPreSelector, SegmentSelector segmentSelector,
-        List<SegmentPruner> segmentPruners, InstanceSelector instanceSelector, int lastUpdateIdealStateVersion,
-        int lastUpdateExternalViewVersion, @Nullable TimeBoundaryManager timeBoundaryManager,
+    RoutingEntry(String tableNameWithType, String externalViewPath, String idealStatePath,
```

Comment:
(minor) Shall we put `idealStatePath` in front of `externalViewPath` to be consistent with the order of `lastUpdateVersion`?

Reply:
sure, I just put them in alphabetical order